documentation/modules/managing/con-broker-config-properties.adoc

```diff
@@ -109,10 +109,11 @@ As an exception, you might want to reduce the settings in a single-broker _test_
 Network threads handle requests to the Kafka cluster, such as produce and fetch requests from client applications.
 Produce requests are placed in a request queue. Responses are placed in a response queue.
 
-The number of network threads should reflect the replication factor and the levels of activity from client producers and consumers interacting with the Kafka cluster.
+The number of network threads per listener should reflect the replication factor and the levels of activity from client producers and consumers interacting with the Kafka cluster.
```

Comment:
@mimaison -- do we we need to say something about prefixing with listener name?

Reply:
I don't think we currently allow to configure this differently for different listeners.

Reply:
It's new in 3.2 so yes we don't have it yet.

Reply:
Okay, thanks